### ISSUE TITLE
Search for literature on common misconceptions in programming intro

### ISSUE THREAD

**Issue by dbosk** [dbosk/intropy#9]

**Comment by celinasoori** [comment]
Hur mycket vill vi ha på detta? Börjat gräva ner mig i ett hål nu nämligen, och jag kommer kunna fortsätta gräva ett bra tag till

**Comment by dbosk** [comment]
Vi vill ju helst ha allt 😉Men inför kriterier. Typ att det ska vara relaterat till python (eller inte Java 😝). Typ att du slutar när de 10 senaste artiklarna inte tillför något nytt missförstånd.Du känner ju vad som är vettigt. Men ha några konkreta kriterier så att någon annan kan göra exakt lika.Du kan ju även börja med att titta på existerande surveys. Och då titta på detaljerna i artiklarna de tar upp.Sent from a low-usability deviceOn 3 Jun 2022 17:16, celinasoori ***@***.***> wrote:

Hur mycket vill vi ha på detta? Börjat gräva ner mig i ett hål nu nämligen, och jag kommer kunna fortsätta gräva ett bra tag till

**Comment by celinasoori** [comment]
Tack, bra! Då fortsätter jag i mitt hål.

Vill du att jag lägger in de artiklar/uppsatser som bekräftar det tidigare redan sagt? Börjat hamna där nu

**Comment by dbosk** [comment]
Kan vara bra att ha med "bekräftas även av X", men inga fler detaljer som abstract etc.

**Comment by dbosk** [comment]
Denna dök upp nu på ITiCSE'22: [Experience Report: Identifying Unexpected Programming Misconceptions with a Computer Systems Approach](https://dl-acm-org.focus.lib.kth.se/doi/10.1145/3502718.3524775).

**Comment by dbosk** [comment]
Ja, om du ändå går igenom dem kan du gärna lägga in dem också. Lägg in på samma ställe då så att det är tydligt att de bekräftar samma sak.Sent from a low-usability deviceOn 3 Jun 2022 17:58, celinasoori ***@***.***> wrote:

Tack, bra! Då fortsätter jag i mitt hål.

Vill du att jag lägger in de artiklar/uppsatser som bekräftar det tidigare redan sagt? Börjat hamna där nu
